django/template/base.py

```diff
@@ -855,6 +859,9 @@ def _resolve_lookup(self, context):
                             else:
                                 raise
         except Exception as e:
+            template_name = getattr(context, "template_name", "unknown")
+            logger.warning("%s - %s", template_name, e)
```

Comment:
Could you also update `logger.warning` to `logger.debug`. You may want to wait for confirmation from a core developer before change it. Thanks!